packages/editor-ui/src/components/NodeCreator/NodeItem.vue

```diff
@@ -21,21 +26,41 @@
 					})
 				}}
 			</div>
+
+			<div :class="$style['draggable-data-transfer']" ref="draggableDataTransfer" />
+			<transition name="node-item-transition">
+				<div
+					:class="$style.draggable"
+					:style="draggableStyle"
+					ref="draggable"
+					v-show="dragging"
+				>
+					<NodeIcon class="node-icon" :nodeType="nodeType" :size="40" :shrink="false" />
+				</div>
+			</transition>
 		</div>
 	</div>
 </template>
 
 <script lang="ts">
 
+import {getNewNodePosition, NODE_SIZE} from '@/views/canvasHelpers';
 import Vue from 'vue';
-import { INodeTypeDescription } from 'n8n-workflow';
 
 import NodeIcon from '../NodeIcon.vue';
 import TriggerIcon from '../TriggerIcon.vue';
 
 Vue.component('NodeIcon', NodeIcon);
 Vue.component('TriggerIcon', TriggerIcon);
 
+interface Data {
```

Comment:
nitpick - not sure this type is necessary or helping. If you check types below `this.$data.dragging` has type `any` while `this.dragging` has type `number`. If you remove the return type, end result is the same. 
i would recommend dropping this return type and replacing `this.$data.key` with `this.key` since those are typed.

Reply:
Makes sense. In the end, it looks like the actual culprit for type errors was the `isTrigger` property being defined directly at the top level of the Vue object. This was just me trying to fix issues until I realized what was wrong. Removed the `Data` interface and changed to `this.key`.